configs/centernet/centernet_r18-dcnv2_8xb16-crop512-140e_coco.py

```diff
@@ -133,3 +133,43 @@
 # USER SHOULD NOT CHANGE ITS VALUES.
 # base_batch_size = (8 GPUs) x (16 samples per GPU)
 auto_scale_lr = dict(base_batch_size=128)
+
+tta_model = dict(
+    type='DetTTAModel',
+    tta_cfg=dict(nms=dict(type='nms', iou_threshold=0.5), max_per_img=100))
+
+tta_pipeline = [
+    dict(
+        type='LoadImageFromFile',
+        to_float32=True,
+        file_client_args=dict(backend='disk')),
+    dict(
+        type='TestTimeAug',
+        transforms=[
+            [
+                # ``RandomFlip`` must be placed before ``RandomCenterCropPad``,
+                # otherwise bounding box coordinates after flipping cannot be
```

Comment:
I don't understand why? can you explain?

Reply:
If we want to recover the flipped result after padding, we need to know how many pixels are padding in the upper left corner of the image. This problem can be avoided by using flip first and then padding.